examples/libf3d/python/multi_geom.py

```diff
@@ -0,0 +1,26 @@
+import sys
+import f3d
+from pathlib import Path
+
+if __name__ == "__main__":
+    if len(sys.argv) > 2:
+        sys.exit(1)
+
+    # Load static plugins
+    f3d.Engine.autoload_plugins()
+
+    # Create a native window engine
+    eng = f3d.Engine(f3d.Window.NATIVE)
+
+    # Load all files from provided directory as geometries
+    files = [f for f in Path(sys.argv[1]).iterdir() if f.is_file()]
+    for file in files:
+        eng.loader.load_geometry(str(file))
```

Comment:
```suggestion
    for file in files:
        try:
            eng.loader.load_geometry(str(file))
        except RuntimeError as e:
            print(e) # or something
```
would avoid crashing the whole program if a file is unsupported (maybe f3d should raise a more helpful `IOError` here actually?)

Reply:
`load_failure_exception` already exists

Reply:
> `load_failure_exception` already exists

indeed `load_failure_exception : public exception`, and `pybind` maps c++'s `exception` to python's `RuntimeError`.
May be worth investigating https://pybind11.readthedocs.io/en/stable/advanced/exceptions.html#registering-custom-translators so we can map it to `IOError` which is more specific

Reply:
alright, definitely an improvement to the bindings to consider